ios/FluentUI/Core/Theme/Tokens/AliasTokens.swift

```diff
@@ -423,6 +423,9 @@ public final class AliasTokens {
         case brandStroke1
         case brandStroke1Pressed
         case brandStroke1Selected
+        // Shared colors
+        case redForeground1
```

Comment:
Are there more than yellow and red foreground colors that should be added as part of this change?

Reply:
Not for label but most likely we will need to add a few more for BadgeView and NotificationView.

Reply:
Would have been ideal to add all the new colors in a separate PR, but I'd go ahead and add them here so they're ready to go for future changes.

Reply:
I thought we discussed creating shared colors alias tokens as we encounter them on a per control basis. It would save us some time and prevent us from creating tokens that will be unused.

Reply:
I don't believe that should be the case: all tokens must be part of our library for clients to consume, so there will never be "unused tokens".